website/docs/d/epg_def.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+subcategory: "L4-L7 Services"
+layout: "aci"
+page_title: "ACI: aci_epg_def"
+sidebar_current: "docs-aci-data-source-epg_def"
+description: |-
+  Data source for ACI EPg Def
+---
+
+# aci_e_pg_def #
+
+Data source for ACI EPg Def
+
+## API Information ##
+
+* `Class` - vnsEPgDef
+* `Distinguished Name` - uni/tn-{name}/GraphInst_C-[{ctrctDn}]-G-[{graphDn}]-S-[{scopeDn}]/NodeInst-{name}/LegVNode-{name}/EPgDef-{name}
```

Comment:
Please used different values instead of "name" like(tenant_name, etc).

It is good to have if the "GUI Information" is available.